products/healthcare/api.yaml

```diff
@@ -540,3 +540,76 @@ objects:
         'Creating a HL7v2 Store':
           'https://cloud.google.com/healthcare/docs/how-tos/hl7v2'
       api: 'https://cloud.google.com/healthcare/docs/reference/rest/v1/projects.locations.datasets.hl7V2Stores'
+  - !ruby/object:Api::Resource
+    name: 'ConsentStore'
+    kind: "healthcare#consentStore"
+    base_url: 'projects/{{project}}/locations/{{location}}/consentStores?consentStoreId={{consent_id}}'
+    self_link: 'projects/{{project}}/locations/{{location}}/consentStores/{{consent_id}}'
+    update_verb: :PATCH
+    update_mask: true
+    min_version: beta
+    description: |
+      The Consent Management API is a tool for tracking user consents and the documentation associated with the consents. 
+    iam_policy: !ruby/object:Api::Resource::IamPolicy
+      method_name_separator: ':'
+      fetch_iam_policy_verb: :POST
+      parent_resource_attribute: 'consent_id'
+      import_format: ["projects/{{project}}/locations/{{location}}/consentStores/{{consent_id}}", "{{consent_id}}"]
+      base_url: projects/{{project}}/locations/{{location}}/consentStores/{{consent_id}}
+      self_link: projects/{{project}}/locations/{{location}}/consentStores/{{consent_id}}
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'name'
```

Comment:
This value should already be exposed as the `id` of the resource. I think we can change `name` to be the input field rather than `consent_id`, which would fit with our other resources better